package/imx-mkimage/imx-mkimage.mk

```diff
@@ -4,26 +4,25 @@
 #
 ################################################################################
 
-IMX_MKIMAGE_VERSION = lf-5.10.72_2.2.0
+IMX_MKIMAGE_VERSION = lf-6.1.22-2.0.0
 IMX_MKIMAGE_SITE = $(call github,nxp-imx,imx-mkimage,$(IMX_MKIMAGE_VERSION))
 IMX_MKIMAGE_LICENSE = GPL-2.0+
-IMX_MKIMAGE_LICENSE_FILES = COPYING
+IMX_MKIMAGE_LICENSE_FILES = LICENSE
 HOST_IMX_MKIMAGE_DEPENDENCIES = host-zlib
 
 ifeq ($(BR2_PACKAGE_FREESCALE_IMX_PLATFORM_IMX8M)$(BR2_PACKAGE_FREESCALE_IMX_PLATFORM_IMX8MM)$(BR2_PACKAGE_FREESCALE_IMX_PLATFORM_IMX8MN)$(BR2_PACKAGE_FREESCALE_IMX_PLATFORM_IMX8MP),y)
 # i.MX8M needs a different binary
 define HOST_IMX_MKIMAGE_BUILD_CMDS
 	$(HOST_MAKE_ENV) $(MAKE) $(HOST_CONFIGURE_OPTS) \
 		CFLAGS="$(HOST_CFLAGS) -std=c99" \
-		-C $(@D)/iMX8M -f soc.mak mkimage_imx8
+		-C $(@D) SOC=iMX8M iMX8M/mkimage_imx8
 endef
 define HOST_IMX_MKIMAGE_INSTALL_CMDS
 	$(INSTALL) -D -m 755 $(@D)/iMX8M/mkimage_imx8 $(HOST_DIR)/bin/mkimage_imx8
 	$(INSTALL) -D -m 755 $(@D)/iMX8M/mkimage_fit_atf.sh $(HOST_DIR)/bin/mkimage_fit_atf.sh
 	$(INSTALL) -D -m 755 $(@D)/iMX8M/print_fit_hab.sh $(HOST_DIR)/bin/print_fit_hab.sh
 endef
 else
-# i.MX8 and i.MX8X
```

Comment:
why remove the comment?

Reply:
It's hard to update the comment to keep track of all supported SOC's except i.MX8M, i.MX8MM, i.MX8MN and i.MX8MP.